index.js

```diff
@@ -2,30 +2,15 @@
  * Module dependencies.
  */
 
+var conceal = require('conceal');
```

Comment:
Oh... yeah it should be. I was using it to mask the token earlier before I refactored it. I'll add that back in.